source/community/reactnative/src/components/containers/columnset.js

```diff
@@ -48,8 +50,8 @@ export class ColumnSet extends React.PureComponent {
 		const payload = this.payload;
 
 		var columnSetContent = (
-			<ContainerWrapper style={{ flex: this.payload.columns.length}} json={payload} containerStyle={this.props.containerStyle}>
-				<ElementWrapper json={payload} style={styles.defaultBGStyle} isFirst={this.props.isFirst}>
+			<ContainerWrapper style={{ flex: this.payload.columns.length }} json={payload} containerStyle={this.props.containerStyle}>
+				<ElementWrapper json={payload} isColumnSet={true} style={styles.defaultBGStyle} isFirst={this.props.isFirst}>
```

Comment:
Existing prop `json` contains the entire element payload which has the element type info. If so, no need for such additional prop that pass the element type info. This check can go inside the ElementWrapper implementation with `this.props.json.type`.

Reply:
Done. Consumed` this.props.json.type`